app/views/occupation_standards/index.html.erb

```diff
@@ -23,11 +23,11 @@
 
 <section aria-labelledby="occupation-title" class="bg-seafoam-50 dark:bg-gray-900">
     <div class="mx-auto max-w-screen-xl py-8 px-4 sm:py-16 lg:px-6">
-        <% if @occupation_standards_search.search_term[:search_term].present? %>
+        <% if @occupation_standards_search.q.present? %>
```

Comment:
Now I feel like I'm confused. I thought this would look like: `@occupation_standards_search.search_term[:q].present?`

Reply:
That way we can have `q` as a general search query term, but we can also start adding `title`, `onet`, etc. as more specific filters?

Reply:
Search term is the query from the UI. Nesting q inside search_term may not make a lot of sense since we won't have multiple search terms

Reply:
What about adding filters? Won't we use the same Query interface?

Reply:
We'll use the same, but not sure if we have defined how we are going to implement the filters. We can use q for title, and then specific fields for the filters 🤔

Reply:
My vote is either:

```
@occupation_standards_search = OccupationStandardQuery::Container.new(search_term: search_term_params)

def search_term_params
 {q: params[:q]}
end
```

or

```
@occupation_standards_search = OccupationStandardQuery::Container.new(search_term: params[:q])
```

But I think my least favorite is just `q` running around by itself somewhat ill-defined.

Reply:
And I think my vote is really the top one, so that we can pass other parameters for filters (thinking ahead to filtering by the `national_standard_type` field by itself).